src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.nuspec

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<package>
+  <metadata minClientVersion="2.5">
+    <id>Microsoft.Diagnostics.Tracing.TraceEvent</id>
+    <version>0.0.0</version>
```

Comment:
:memo: This along with `$version$` are automatically populated from the `[AssemblyInformationalVersion]` attribute value.